config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -271,6 +271,6 @@ periodics:
   annotations:
     testgrid-num-failures-to-alert: '2'
     testgrid-alert-stale-results-hours: '48'
-    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind
+    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind, sig-release-master-blocking
     testgrid-tab-name: conformance-ga-only
     testgrid-alert-email: liggitt@google.com
```

Comment:
Can we have a non-human alert group here?
If it's on master-blocking kubernetes-release-team@googlegroups.com ideally is also one of the alerted parties.

Reply:
changed to sig-release and sig-architecture (in lieu of a conformance-specific list)